src/utils/registerError.ts

```diff
@@ -0,0 +1,18 @@
+export function registerError(error: Error, message?: string, type = 'error') {
```

Comment:
Decided not to add additional logic to pass rum logger from internal version since we already have all rum styles defined in embedded version